mundane-csv/src/main/scala/com/ambiata/mundane/csv/SimpleCsv.scala

```diff
@@ -28,14 +28,23 @@ case class SimpleCsv(delimiter: Char, quoteCharacter: Option[Char], escapeCharac
 
   def parse(line: String): String \/ List[String] =
     \/.fromTryCatchNonFatal {
-      if (line.isEmpty) List("")
+      if (line.isEmpty) emptyLineDefault
```

Comment:
The situation I want to address is the lack of good enough specification for CSV. I've chosen a bunch of test cases which mandate that `parse("") === List("")`, so this is what I have encoded in my test suite and in the parsers. On the other hand in some cases what we want is `parse("") === Nil` (cf `ListParser.delimitedValues`). This is why this can be changed.